### PR TITLE
Wallet component added.

### PR DESCRIPTION
**Pull request by simonburns77**

### CODE REVIEW

**Comment by recursion** [comment]
Looking great Simon! You are definitely on the right track here. 

I see this does still need to be pulled out into its own module folder/file/, and all inline css needs to be removed. (use classes/ids and css (or sass) in the module's .scss file) otherwise I think it looks ok at first glance.

Im not sure if you were there when we walked through hooking a module into ui-router? Dick, Amy, or myself can definitely walk you through that if you are want.

**Comment by mnaamani** [comment]
caution, I see several api keys (secrets) in app.js  (for parse and block.io)
Don't use those accounts except for demo purposes from now on.

File: client/app/app.html [hunk 1]
```diff
@@ -3,3 +3,78 @@
...
+	    </div>
+	 </div>
+
+<!--   <div class="chart" ng-controller="chartCtrl" ng-model="">
+    <chartist class="ct-chart ct-golden-section" chartist-data="Chartist.barData" chartist-chart-type="Bar" chartist-chart-options="Chartist.barOptions" chartist-responsive-options="Chartist.barResponsiveOptions">
+      
+    </chartist>
+  </div> -->
+
+	<div style="float:right" class="execution" ng-controller="executionCtrl" >
```
Comment (recursion) [inline client/app/app.html]: This inline styling needs to be removed from this file.

File: client/app/app.html [hunk 2]
```diff
@@ -3,3 +3,78 @@
...
+
+  <div style="float:right" class="wallet" ng-controller="walletCtrl">
+    <div class="close"> <h4>Wallet Address:</h4>
+        <h4> {{ showWallet }}</h4>
+    </div>
+    <div style="position: absolute"> <h3></h3></div>
+    <div style="margin-top: 30px">
+      Balance
+    </div>
+    <div style="margin-top: 20px">
```
Comment (recursion) [inline client/app/app.html]: All of the inline styling in this file needs to go! 
It should all be handled through the CSS.

File: client/app/app.js [hunk 3]
```diff
@@ -155,4 +155,55 @@
     }])
   //---------------------------------------------------------
 
+   app.controller('walletCtrl', ['$scope',
```
Comment (recursion) [inline client/app/app.js]: This code needs to be moved into its own module folder.
Reach out if you would like another walk through on how to do this.

File: client/app/app.scss [hunk 4]
```diff
@@ -40,6 +40,15 @@ h2, h3, h4, h5, h6 {
   width: 300px;
 }
 
+.wallet {
```
Comment (recursion) [inline client/app/app.scss]: This code should be in the wallet module css file, not inside the main application css.

**Pull request merged**